packages/contracts-bedrock/scripts/ChainAssertions.sol

```diff
@@ -38,26 +38,38 @@ library ChainAssertions {
         ResourceMetering.ResourceConfig memory dflt = Constants.DEFAULT_RESOURCE_CONFIG();
         require(keccak256(abi.encode(rcfg)) == keccak256(abi.encode(dflt)));
 
-        checkSystemConfig({ _contracts: _prox, _cfg: _cfg, _isProxy: true });
+        checkSystemConfig({ _contracts: _prox, _cfg: _cfg, _vm: _vm, _isProxy: true });
         checkL1CrossDomainMessenger({ _contracts: _prox, _vm: _vm, _isProxy: true });
-        checkL1StandardBridge({ _contracts: _prox, _isProxy: true });
+        checkL1StandardBridge({ _contracts: _prox, _vm: _vm, _isProxy: true });
         checkL2OutputOracle({
             _contracts: _prox,
             _cfg: _cfg,
             _l2OutputOracleStartingTimestamp: _l2OutputOracleStartingTimestamp,
+            _vm: _vm,
             _isProxy: true
         });
-        checkOptimismMintableERC20Factory({ _contracts: _prox, _isProxy: true });
-        checkL1ERC721Bridge({ _contracts: _prox, _isProxy: true });
-        checkOptimismPortal({ _contracts: _prox, _cfg: _cfg, _isProxy: true });
-        checkProtocolVersions({ _contracts: _prox, _cfg: _cfg, _isProxy: true });
+        checkOptimismMintableERC20Factory({ _contracts: _prox, _vm: _vm, _isProxy: true });
+        checkL1ERC721Bridge({ _contracts: _prox, _vm: _vm, _isProxy: true });
+        checkOptimismPortal({ _contracts: _prox, _cfg: _cfg, _vm: _vm, _isProxy: true });
+        checkProtocolVersions({ _contracts: _prox, _cfg: _cfg, _vm: _vm, _isProxy: true });
     }
 
     /// @notice Asserts that the SystemConfig is setup correctly
-    function checkSystemConfig(Types.ContractSet memory _contracts, DeployConfig _cfg, bool _isProxy) internal view {
+    function checkSystemConfig(
+        Types.ContractSet memory _contracts,
+        DeployConfig _cfg,
+        Vm _vm,
+        bool _isProxy
+    )
+        internal
+        view
+    {
         console.log("Running chain assertions on the SystemConfig");
         SystemConfig config = SystemConfig(_contracts.SystemConfig);
 
+        bytes32 initialized = _vm.load(address(config), bytes32(0));
+        require(initialized != 0);
```

Comment:
We don't currently do this, but I have a strong preference for using the error strings—without them, if a `require` is hit, it's much harder to track down which check failed. I'd suggest adding this for all the new require statements in this PR

I like doing this with incrementing numbers to the statements, in increments of 100, which leaves room to add future statements in between without a huge diff

```suggestion
        require(initialized != 0, "100");
```